### PR TITLE
MSP OSD: more notes on panel items

### PR DESCRIPTION
**Pull request by yaapu**

### CODE REVIEW

File: common/source/docs/common-msp-osd-overview-4.2.rst [hunk 1]
```diff
@@ -43,64 +43,63 @@ OSD Panel Items
...
 +---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| OSDn_GSPEED   | This item shows ground speed unless ``OSDn_ASPEED_EN`` = 1 in which case it will show true airspeed if an airspeed sensor is present or estimated airspeed otherwise. The position on screen is set by ``OSDn_GSPEED_X`` and ``OSDn_GSPEED_Y`` regardless of the value of ``OSDn_ASPEED_EN``         |
+| OSDn_GSPEED   | This item shows ground speed unless OSDn_ASPEED_EN = 1 in which case it will show true airspeed if an airspeed sensor is present or estimated airspeed otherwise. The position on screen is set by OSDn_GSPEED_X and OSDn_GSPEED_Y irregardless of the value of OSDn_ASPEED_EN                       |
```
Comment (Hwurzburg) [inline common/source/docs/common-msp-osd-overview-4.2.rst:74]: my  practice is to always highlite a parameter name, if possible with it linked....when talking about a muli-instance one I use bold (double back-ticks)
also "irregardless" while defined in English, is a non-standard word...use "regardless" instead

File: common/source/docs/common-msp-osd-overview-4.2.rst [hunk 2]
```diff
@@ -43,64 +43,63 @@ OSD Panel Items
...
-| OSDn_MESSAGE  | This will display status text messages as rolling text. Status text messages will be hidden after a couple seconds and the panel will show the current flightmode. If ``OSDn_WIND_EN`` is set to 1, this item also displays wind info next to the current flight mode as a rotating arrow and speed. |
+| OSDn_MESSAGE  | This will display status text messages as rolling text. Status text messages will be hidden after a couple seconds and the panel will show the current flightmode. If OSDn_WIND_EN is set to 1, this item will also display wind info next to the current flight mode as a rotating arrow and speed. |
 +---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (Hwurzburg) [inline common/source/docs/common-msp-osd-overview-4.2.rst:85]: same here regading bolding of the params...which requires modification of the wording to fit and prevent having to lengthen every other row in the table

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
thanks, needs a few changes

**Comment by yaapu** [comment]
ok I'll do

**Comment by Hwurzburg** [comment]
no need, I incorporated them...need them for the existing wiki

**Pull request closed without merging**